contracts/prize-strategy/PrizeStrategyBuilder.sol

```diff
@@ -10,6 +10,21 @@ import "../token/ControlledTokenProxyFactory.sol";
 import "../external/compound/CTokenInterface.sol";
 
 contract PrizeStrategyBuilder is Initializable {
+  using SafeMath for uint256;
+
+  struct Config {
+    CTokenInterface cToken;
+    uint256 prizePeriodSeconds;
+    bytes ticketName;
+    bytes ticketSymbol;
+    bytes sponsorshipName;
+    bytes sponsorshipSymbol;
```

Comment:
We could optimize for gas by using static types like bytes16 or bytes32, and also tightly packing the vars together here, with all uint256 vars at the top, followed by bytes32 (names) and bytes16 (symbols)

Reply:
These *need* to be arrays so that users can pass any value in.